goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/GoodDeeds.java

```diff
@@ -1,5 +1,7 @@
 package com.goodpeople.gooddeeds.Model;
 
+import androidx.annotation.Nullable;
```

Comment:
what is this?

Reply:
No idea, must have been imported during implementation... it's unused though, so I'll remove it.